app/src/helpers/requests/getPresignedUrlForZip.ts

```diff
@@ -11,30 +17,85 @@ type Args = {
     docReferences?: Array<string>;
 };
 
-type GetPresignedUrl = {
-    data: string;
+type GetRequestArgs = {
+    jobId: string;
+    baseUrl: string;
+    baseHeaders: AuthHeaders;
+};
+
+const ThreePendingErrorMessage =
+    'Document Manifest api call responded with "Pending" for 3 attempts.';
```

Comment:
Maybe a different message, "Failed to initiate download"?

Reply:
thanks! fixed this just now